src/functions-and-arrays.js

```diff
@@ -1,41 +1,151 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  }
+  return num2;
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arrWords) {
+  if (arrWords.length === 0) {
+    return null;
+  } else if (arrWords.length !== 0) {
```

Comment:
This check here is redundant, since you are already checking if the length of the array is equal to 0 on line 13. This if block will be read in case it is not, automatically. 😄